packages/embark-utils/src/index.js

```diff
@@ -90,13 +90,18 @@ function runCmd(cmd, options, callback) {
   });
 }
 
+function copyToClipboard(text) {
+  require('clipboardy').writeSync(text);
```

Comment:
Let's pull out this `require` in case we use `copyToClipboard` multiple times? The majority of the time, we use it once, so may as well load it when utils is loaded.

Reply:
Requiring multiple times doesn't cost much since it gets cached, but might as well. It's probably small.